### ISSUE TITLE
running topologyUpdater.sh shows error: double check custom peers

### ISSUE THREAD

**Issue by edridudi** [cardano-community/guild-operators#880]
When running topologyUpdater.sh script after Upgrading to Cardano node 1.26.1 and updating topologyUpdater.sh:

the following error is showing:
ERROR: Invalid IPv4 or IPv6 address '<IP>'. Please double check CUSTOM_PEERS definition

my topologyUpdater.sh held CUSTOM_PEERS Field with the manner that is shown in the description:

CUSTOM_PEERS="<IP_BP>,6000,1|<IP,RELAY1>,6000,1|relays-new.cardano-mainnet.iohk.io,3001,2"

looking in the script I saw that the condition for IP verification is wrong:

should be "&&" instead of "||"
also there needs to be verification of Nodes which are written as FQDN (hostnames)
 
As a workaround, I suggest adding another method for env file:

```
isValidHost() {
  fqdn=$1
  result=`echo $fqdn | grep -P '(?=^.{1,254}$)(^(?>(?!\d+\.)[a-zA-Z0-9_\-]{1,63}\.?)+(?:[a-zA-Z]{2,})$)'`
  if [[ -z "$result" ]]; then
    return 1
  else
    return 0
  fi
}
```

and for topologyUpdater.sh replace this line:
```
if ! isValidIPv4 "${addr}" || ! isValidIPv6 "${addr}";  then echo "ERROR: Invalid IPv4 or IPv6 address '${addr}'. Please double check CUSTOM_PEERS definition"; exit 1
```

with this line:
```
if ! isValidIPv4 "${addr}" && ! isValidIPv6 "${addr}" && ! isValidHost "${addr}";  then echo "ERROR: Invalid IPv4 or IPv6 address '${addr}'. Please double check CUSTOM_PEERS definition"; exit 1
```

that change should be reviewed of course.

Best Regards.

**Comment by Scitz0** [comment]
Yes, the issue is identified and we have a PR to fix it basically done, just need a final review.
#879

**Comment by Scitz0** [comment]
Fix now pushed to master, #879 and #881
